language/src/ceylon/language/Sequence.ceylon

```diff
@@ -287,8 +287,13 @@ shared sealed interface Sequence<out Element=Anything>
         
         first => outer.first;
         size => outer.size*times;
-        rest => sublistFrom(1).sequence(); //TODO!
-        
+        rest => if (times==1)
+                    then outer.rest
+                    else let (tail = Repeat(times-1))
+                         if (outer.longerThan(1))
+                             then outer.rest.append(tail)
+                             else tail;
+
         getFromFirst(Integer index)
```

Comment:
The old code does need to be improved, but why didn't it work??? In fact, the following *does* work on the JS and Dart backends on `1.3.2`:

```ceylon
assert ((0..1).repeat(2).rest.sequence() == [1,0,1]);
```

And I probably need more coffee before asking this... but what is `Repeat` in `Repeat(times-1)` referencing? Is it `outer`'s `Repeat` (what we want for this code), or is `Repeat` inherited (which would repeat the repeat)?

We probably want `outer.Repeat` to be safe, even though `Repeat` is not `shared`... The backends seem to differ on this too.